### PR TITLE
Prettier leaderboard

### PR DESCRIPTION
**Pull request by itsisak**
Last update:
Screenshot 2023-04-21 at 00 13 23

### Thought this looked a bit cleaner (EDIT: added positions)

Skjermbilde 2023-04-20 kl  13 22 00

### I tried adding animation aswell, tell me what you think, could slow it down maybe

https://user-images.githubusercontent.com/45620425/233347467-53304c3e-9586-41c5-832f-fbbef55701ff.mov

This is going to have merge conflicts with the other PR thats out now, but I will fix that

### CODE REVIEW

File: client/src/components/leaderboard.tsx [hunk 1]
```diff
@@ -15,16 +19,19 @@ const Leaderboard = ({scores}:LeaderboardProps) => {
 
     return (
         <div className={"leaderboard"}>
-            {scores.map(score => <Score key={score.name} name={score.name} score={score.score} />)}
+            {scores.map(score => <Score key={score.name} name={score.name} score={score.score} pos={scores.indexOf(score) + 1} />)}
```
Comment (haukkagu) [inline client/src/components/leaderboard.tsx:22]: I'm pretty sure you can do it like this instead: `{scores.map((score, index) => <Score key={score.name} name={score.name} score={score.score} pos={index + 1} />)}`. It looks a little bit cleaner imo :)

File: client/src/routes/home/index.tsx [hunk 2]
```diff
@@ -1,25 +1,29 @@
 import Leaderboard from "src/components/leaderboard";
 
-import styles from "../../app.css";
+import "../../app.css";
 
 const scores = [
   {
-    name: "Jenny",
+    name: "JENNY",
```
Comment (haukkagu) [inline client/src/routes/home/index.tsx:7]: Instead of hardcoding the names to uppercase, you can add a `text-transform: uppercase` property in the CSS

**Review by haukkagu — APPROVED** [review]
Very sexy 🍆

File: client/src/components/leaderboard.css [hunk 3]
```diff
@@ -8,15 +8,57 @@
...
+.score p {
+    padding-left: 30px;
+    background: rgb(165, 50, 71);
+    color: #f2f2f2;
+    width: 90%;
+    height: 100%;
+    border-radius: 100px;
+    display: flex;
+    align-items: center;
+    padding-left: 50px;
```
[the remark this answers is not in the document]
Comment (itsisak) [reply client/src/components/leaderboard.css:38]: oops thats an oversight

File: client/src/routes/home/index.tsx (hunk 2 above)
Comment (itsisak) [reply client/src/routes/home/index.tsx:7]: Yea I agree thats a better solution, could also do `.toUpperCase()` in the tsx file

File: client/src/components/leaderboard.tsx (hunk 1 above)
Comment (itsisak) [reply client/src/components/leaderboard.tsx:22]: oh yea, thats much better! Thanks

**Pull request merged**